### ISSUE TITLE
BC Transit has overhauled how it structures its data

### ISSUE THREAD

**Issue by canassist** [TransitFeeds/TransitFeeds-Public#154]
Previously, BC Transit exposed a single URL (http://bctransit.com/servlet/bctransit/data/GTFS.zip), which is what TransitFeeds checks. The GTFS linked to by this URL actually contained data for _12_ cities/regions including Kelowna, Victoria, and Whistler.

As of Friday, June 17, BC Transit has updated its parser. It now splits out the various cities/regions, and offers each via separate URLs. The home page for these feeds is http://bctransit.com/victoria/footer/open-data. Each region's data has a URL like this (for Victoria): http://bctransit.com/servlet/bctransit/data/GTFS - Victoria

In summary: the previous BC Transit GTFS URL which TransitFeeds uses is broken. New, per-region URLs are now available at http://bctransit.com/victoria/footer/open-data.

**Comment by mldickens** [comment]
Thanks. I'm in the process of adding these. If they wanted to provide separate feeds for regions, they should have done that AND provided one feed with all their service.

**Comment by canassist** [comment]
I think it is fair to suggest that to BC Transit, which I will do.

**Comment by canassist** [comment]
Thank you for adding the newly split agencies. This comment is to report a problem with two of them.

The GTFS feeds for Central Fraser Valley and Chilliwack are available for direct download, i.e., when manually browsing feeds. However, their respective agency IDs (bc-transit/686 and bc-transit/687) do not appear to be uniformly available through the API. That is, although I have successfully clicked their "Download using API" links while logged in, the list of transit agencies returned by programmatically calling getFeeds() for each value in getLocations() does not include these agencies. (The returned list included bc-transit/688-697, inclusive, but not 686 or 687.)

**Comment by HendX** [comment]
As the feeds are now updated I'll close this ticket.
